tests/smartclone_test.go

```diff
@@ -60,17 +60,15 @@ var _ = Describe("[vendor:cnv-qe@redhat.com][level:component]SmartClone tests",
 		})
 	})
 
-	Describe("Verify DataVolume Smart Cloning - Negative flow", func() {
-		It("verify inapplicable smart-clone dv", func() {
-			if !f.IsSnapshotStorageClassAvailable() {
-				Skip("Test not available in openshift")
-			}
+	FDescribe("Verify DataVolume Smart Cloning - Check regular clone works", func() {
+		It("Verify inapplicable smart-clone dv", func() {
+			smartApplicable := f.IsSnapshotStorageClassAvailable()
 			sc, err := f.K8sClient.StorageV1().StorageClasses().Get(f.SnapshotSCName, metav1.GetOptions{})
```

Comment:
Need to skip here if !smartApplicable otherwise this will fail.

Reply:
> Need to skip here if !smartApplicable otherwise this will fail.
It's a negative test, so we skip in line 70 when smart clone is applicable - to test regular cloning.

Reply:
Right, but line 66 is failing because the storage class doesn't exist.

Reply:
> Right, but line 66 is failing because the storage class doesn't exist.

Oh right, need to check for error as we do in cloner_test.